webapp/channels/src/utils/desktopAPI.ts

```diff
@@ -0,0 +1,288 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import semver from 'semver';
+
+import {isDesktopApp} from 'utils/user_agent';
+
+// TODO: Temporary typings
+import type {DesktopAPI} from '../../../../../desktop/src/types/api/main';
```

Comment:
This needs to change before we merge, as we should be getting these types from an npm package.

Reply:
@hmhealey Might need your help with this one.